docker-start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p /data/db
```

Comment:
I wonder if this folder can be made during the docker build.

Reply:
It won't matter as `mkdir` is not going to cost any significant time.

Reply:
`O(1)` vs `O(n)` some would say...